zeppelin-web/src/components/noteName-import/note-import-dialog.html

```diff
@@ -11,71 +11,70 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<div id="noteImportModal" class="modal fade" role="dialog" tabindex="-1"
+     data-backdrop="static" data-keyboard="false">
 
-  <div id="noteImportModal" class="modal fade" role="dialog"
-       tabindex="-1" data-backdrop="static" data-keyboard="false">
-    <div class="modal-dialog" >
+  <div class="modal-dialog">
+    <div class="modal-content" id="NoteImportCtrl" ng-init="NoteImportInit">
 
-      <!-- Modal content-->
-      <div class="modal-content" id="NoteImportCtrl" ng-init="NoteImportInit">
-        <div class="modal-header">
-          <!-- close button -->
-          <button type="button" class="close" data-dismiss="modal" ng-click="noteimportctrl.resetFlags()">&times;</button>
+      <!-- modal-header -->
+      <div class="modal-header modal-header-import">
+        <button type="button" class="close" data-dismiss="modal"
+                ng-click="noteimportctrl.resetFlags()">&times;</button>
+        <h4 class="modal-title">Import New Note</h4>
+      </div>
 
-          <h4 class="modal-title">Import new note</h4>
+      <!-- modal-body -->
+      <div class="modal-body modal-body-import">
+        <div class="form-group">
+          <label for="noteImportName">Import As</label>
+          <input class="form-control note-name-input" id="noteImportName"
+                 placeholder="Insert Note Name" type="text"
+                 ng-model="note.noteImportName" />
         </div>
-        <div class="modal-body">
 
-          <div class="form-group">
-            <label for="noteImportName">Import AS</label>
-            <input placeholder="Note name" type="text" class="form-control" id="noteImportName"
-                   ng-model="note.noteImportName" />
-          </div>
-          <div class="form-group">
-            <label for="fileSizeLimit">JSON file size cannot exceed {{maxLimit}} MB</label>
-          </div>
+        <div class="form-group" ng-show="note.errorText">
+          <div class="alert alert-danger">{{note.errorText}}</div>
+        </div>
 
-          <div class="form-group" ng-show="note.errorText">
-            <div class="alert alert-danger">{{note.errorText}}</div>
-          </div>
+        <label>JSON file size cannot exceed {{maxLimit}} MB</label>
```

Comment:
How about adding space of this label (padding-top) like this?
for example (padding-top:15px):

![image](https://cloud.githubusercontent.com/assets/8110458/26296174/70efc2a2-3f08-11e7-94c8-b3d5b3143959.png)